examples/is-prime/README.md

```diff
@@ -0,0 +1,14 @@
+# is-prime 
+
+## Compile
+
+```bash
+cargo miden build --release
+```
+
+## Run
+
+```bash
+midenc run target/miden/release/is_prime.masp --inputs inputs.toml
+```
```

Comment:
This is just executes the program (rather than execute & prove), right?

Reply:
Yes, `midenc run` only executes the program and does not generate a proof. Since comparing our performance to other VMs would be a frequent case, should we add a command to `midenc` to execute and generate a proof?

Reply:
We could do this, but I wonder if it may be better to just add support for working with `.masp` files in `miden-vm`. It should be simple enough and wouldn't require re-implementing a bunch of other functionality here (e.g., `analyze`, `debug`, etc. commands).

Reply:
Yeah, my preference would be to support Miden packages in the VM proper, the need for `midenc run` is basically one of temporary necessity.